src/panels/calendar/ha-big-calendar.js

```diff
@@ -0,0 +1,80 @@
+import '@polymer/app-layout/app-header-layout/app-header-layout.js';
+import '@polymer/app-layout/app-header/app-header.js';
+import '@polymer/app-layout/app-toolbar/app-toolbar.js';
+import '@polymer/paper-dropdown-menu/paper-dropdown-menu.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-listbox/paper-listbox.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import '@vaadin/vaadin-date-picker/vaadin-date-picker.js';
+
+/* polymer-lint disable */
+import React from 'react'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved
```

Comment:
Instead of this, add this to `.eslintrc-hound.json` under the settings key:

```
    "import/resolver": {
      "webpack": {
        "config": "webpack.config.js"
      }
    }
```

Reply:
I still have an issue with that:
```
± node_modules/eslint/bin/eslint.js src/panels/calendar/

/home/tcohen/perso/gits/home-assistant/home-assistant-polymer/src/panels/calendar/ha-big-calendar.js
  14:19  error  '/home/tcohen/perso/gits/home-assistant/home-assistant-polymer/node_modules/preact-compat/dist/preact-compat.es.js' imported multiple times  import/no-duplicates
  15:24  error  '~/perso/gits/home-assistant/home-assistant-polymer/node_modules/preact-compat/dist/preact-compat.es.js' imported multiple times  import/no-duplicates

✖ 2 problems (2 errors, 0 warnings)
```